src/Piece/PieceForm/PieceForm.js

```diff
@@ -97,6 +99,17 @@ const PieceForm = ({
     receivingStatus,
   } = formValues;
 
+  useEffect(() => {
+    if (!id && formValues?.format === PIECE_FORMAT.electronic) {
```

Comment:
this hook will change the form on every update for the electronic piece. I'd suggest handling the piece format field change and put that logic there

Reply:
updated